src/tentaclio/clients/ftp_client.py

```diff
@@ -89,18 +89,42 @@ def scandir(self, **kwargs) -> Iterable[fs.DirEntry]:
         """Scan the connection url to create dir entries."""
         base_url = f"ftp://{self.url.hostname}:{self.port}{self.url.path}/"
         entries = []
-        for mlsd_entry in self.conn.mlsd(self.url.path, facts=["type"]):
-            # https://docs.python.org/3/library/ftplib.html#ftplib.FTP.mlsd
-            file_name = mlsd_entry[0]
-            entry_type = mlsd_entry[1]["type"]
+        try:
```

Comment:
Just for a bit of clarity, could you move content of the try/except to two different functions plz? I think it would make sense as the level of abstraction is similar would make the scandir function a bit more readable. ta!